ui/src/app/app.module.ts

```diff
@@ -44,6 +47,11 @@ import {
   WorkspacesService
 } from 'generated';
 
+// Unfortunately stackdriver-errors-js doesn't properly declare dependencies, so
+// we need to explicitly load its StackTrace dep:
+// https://github.com/GoogleCloudPlatform/stackdriver-errors-js/issues/2
+(<any>window).StackTrace = StackTrace;
```

Comment:
Note to reviewers - this is terrible, but is the least worst way I was able to get things working (preferred to another `<script>` at least). Let me know if you have ideas on improving this.